src/Corporation/Actions.ts

```diff
@@ -90,6 +91,27 @@ export function IssueDividends(corporation: Corporation, rate: number): void {
   corporation.dividendRate = rate;
 }
 
+export function IssueNewShares(corporation: Corporation, amount: number, max: number): number {
```

Comment:
If this is in Corporation/Actions.ts, shouldn't the actual corporation UI use this function? It looks like this is only being used by the new netscript function. Is there a reason both are not using the same function?

Reply:
They do now